CreeDictionary/tests/test_usage_of_morphodict.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+"""
+Integration test of settings.py and morphodict application.
```

Comment:
more like Unit test?

Reply:
This is technically an integration test, because I made two units here: the `morphodict` application itself, and the settings defined in `CreeDictionary.settings`. In my view, this is testing the integration of the two. Does that make sense?